### PR TITLE
Skip collateral factor check on JellyFish

### PR DESCRIPTION
**Pull request by Bushstar**

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -760,7 +760,7 @@ class CRegTestParams : public CChainParams {
     explicit CRegTestParams() {
         strNetworkID = "regtest";
         bool isJellyfish = false;
-        isJellyfish = gArgs.GetBoolArg("-jellyfish_regtest", false);
+        isJellyfish = gArgs.GetBoolArg("-regtest-skip-loan-collateral-validation", false);
```
Comment (Jouzo) [inline src/chainparams.cpp:763]: I'd rather we keep `-jellyfish_regtest` and add an additional flag for skipping `LoanCollateralFactor` validation.
It's confusing now seeing as `isJellyfish` has lots of side effect already. See https://github.com/DeFiCh/ain/blob/4b3c187f590738f44f5574901f1fec233a53c226/src/chainparams.cpp#L884

Comment (prasannavl) [reply src/chainparams.cpp:763]: Agree with Jeremy. Do not touch `jellyfish_regtest`. I'd leave it as it is. The `regtest-skip-loan-collateral-validation` is an additional arg, that is added in addition, not intended to replace jellyfish_regtest.

File: src/masternodes/govvariables/attributes.cpp [hunk 2]
```diff
@@ -1268,6 +1268,11 @@ Res ATTRIBUTES::Apply(CCustomCSView & mnview, const uint32_t height)
                 }
             } else if (attrV0->key == TokenKeys::LoanCollateralFactor) {
                 if (height >= static_cast<uint32_t>(Params().GetConsensus().FortCanningEpilogueHeight)) {
+                    // Skip on JellyFish
+                    if (gArgs.GetBoolArg("-regtest-skip-loan-collateral-validation", false)) {
```
Comment (prasannavl) [inline src/masternodes/govvariables/attributes.cpp:1272]: 1. Add a check to ensure it's first REGTEST network and chain it with the arg after. 
2. Remove the Skip on jellyfish comment.

**Comment by prasannavl** [comment]
Let the PR be reused for now, but do not force push. Branch off and use new PRs.

**Pull request closed without merging**